CHANGELOG.md

```diff
@@ -99,6 +99,7 @@
 * Go: `WATCH` and `UNWATCH` ([#4054](https://github.com/valkey-io/valkey-glide/pull/4054))
 * Java: OTEL fix for script ([#4065](https://github.com/valkey-io/valkey-glide/pull/4065))
 * Go: Add Opentelemetry support of creating spans ([#3932](https://github.com/valkey-io/valkey-glide/pull/3932))
+* Go: Fix API to use errors idiomatically ([#4090](https://github.com/valkey-io/valkey-glide/pull/4090))
```

Comment:
Should be in fixes or in breaking change section?

Reply:
I can move to fixes section. It's not really a breaking change since this is the first GA release. We should not mark changes to a preview release as breaking changes.